test/workbox-google-analytics/sw/test-initialize.mjs

```diff
@@ -147,6 +147,11 @@ describe(`initialize`, function() {
   });
 
   it(`should register GET/POST routes for collect endpoints`, async function() {
+    // Stub out fetch(), since this test is about routing, and doesn't need to
+    // contact the production Google Analytics endpoints.
+    // (This also works around some unresolved issues in Chrome 90+.)
```

Comment:
Do we want to specifically call out the issue in Chrome 90? The stubbing is the right thing to do, it would be sort of a legacy comment that no one other than you and me would understand and that in a couple of years of months even us won't remember what it was about 

Same for bellow.

Reply:
Sure, dropping the comment.